UI/Web/src/assets/langs/en.json

```diff
@@ -1695,7 +1695,8 @@
         "image-scaling-label": "Image Scaling",
         "height": "Height",
         "width": "Width",
-        "manual_width": "Manual Width",
+        "width_override": "Width Override",
```

Comment:
You need to match the naming scheme of the other localization keys. override-width-label and off should be the keys.